src/react/deckgl.js

```diff
@@ -42,6 +40,7 @@ const propTypes = {
   height: PropTypes.number.isRequired,
   layers: PropTypes.arrayOf(PropTypes.instanceOf(Layer)).isRequired,
   effects: PropTypes.arrayOf(PropTypes.instanceOf(Effect)),
+  glOptions: PropTypes.object,
```

Comment:
Nit: Should we add a default prop value for `glOptions: PropTypes.object`, {} ?

Reply:
> Nit: Should we add a default prop value for glOptions: PropTypes.object, {} ?

Done